pkg/controller/master-controller-manager/rbac/sync_resource_test.go

```diff
@@ -1069,6 +1069,241 @@ func TestSyncProjectResourcesNamespaced(t *testing.T) {
 				},
 			},
 		},
+
+		// scenario 2
+		{
+			name:            "scenario 2: a proper set of RBAC Role/Binding is generated for a resource quota",
+			expectedActions: []string{"get"},
+
+			dependantToSync: &kubermaticv1.ResourceQuota{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "project-thunderball",
+					Namespace: "kubermatic",
+					UID:       types.UID("abcdID"),
+					Labels: map[string]string{
```

Comment:
i think we agreed to go with the project owner ref for resource quotas with project subjects.